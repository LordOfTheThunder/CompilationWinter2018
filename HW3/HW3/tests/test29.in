void SomeFunc(int x) {
	int a;
}
void TestPrintI() {
	bool c = true;
	if (not c) {
		int x = 3;
		printi(x);
	}
}

void main() { print("Max is cool"); }
