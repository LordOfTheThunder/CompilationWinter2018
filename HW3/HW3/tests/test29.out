---end scope---
x INT -1
a INT 0
---end scope---
x INT 1
---end scope---
---end scope---
c BOOL 0
---end scope---
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
SomeFunc (INT)->VOID 0
TestPrintI ()->VOID 0
main ()->VOID 0
